src/resources/postcss/views/_loader.pcss

```diff
@@ -0,0 +1,24 @@
+.tribe-events-container {
+	position: relative;
+
+	.tribe-events-view-loader {
+		align-content: center;
+		justify-content: center;
+		display: flex;
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		z-index: 25000;
+		background: rgba( 255, 255, 255, 0.6 );
+
+		&.tribe-hidden {
```

Comment:
This is OK for now, but can we not wrap this in `.tribe-events-view-loader`? I think that if this is a common occurrence, we should use a common class, like `.tribe-common-a11y-hidden` or something similar.

Reply:
Perfecto, using that.

Reply:
Sorry, we do have a class from common that does exactly this: `.tribe-common-a11y-hidden`. However, if we're planning to target this class in the JS, we should probably use something like `.tribe-events-view-loader--hidden` and use `@mixin hidden`.

Reply:
actually scratch that, using `tribe-common-a11y-hidden` is fine with how the JS is set up.